docs/guides/vqa_guide/ansatzes.rst

```diff
@@ -50,4 +49,11 @@ Finally, we'll define the ``_generate_circuit`` function which substitutes param
     :start-at: def _generate_circuit(
     :end-at: return self._circuit.bind(symbols_map)
 
-And that's it! Once the ansatz is defined we can easily subtitute in into an ``EstimationTaskFactory``.
\ No newline at end of file
+And that's it! Once the ansatz is defined we can easily subtitute in into an ``EstimationTaskFactory``.
+
+
+Another important thing to know about our implementation of ansatzes is that it can support symbolic parameters. Conceptually, ansatzes yield themselves nicely be expressed using symbols rather. However, it might come at the price of performance, so we recommend implementing your ansatz using symbolic circuits in two cases:
+
+- if generating circuit is time-consuming. This is the case e.g. when you need to perform time evolution of some Hamiltonian to build the circuit.
```

Comment:
Not clear whats mentioned here.

Reply:
Fixed in https://github.com/zapatacomputing/orquestra-core/pull/60/commits/0b3d2fd9b5438795d3280b6ecad3885360af1ba2